### PR TITLE
Ana levit

### PR DESCRIPTION
**Pull request by anaalamed**
list of students with basic css

### CODE REVIEW

**Comment by davidmeirlevy** [comment]
@anaalamed please resolve your conflicts.

File: david-levy/index.html [hunk 1]
```diff
@@ -1,10 +0,0 @@
-<!DOCTYPE html>
```
Comment (davidmeirlevy) [inline david-levy/index.html]: probably need to rebase better, because you removed this file.

File: shay-geva/index.html [hunk 2]
```diff
@@ -1,45 +0,0 @@
-<!DOCTYPE html>
```
Comment (davidmeirlevy) [inline shay-geva/index.html]: probably need to rebase better, because you removed this file.

File: shay-geva/style.css [hunk 3]
```diff
@@ -1,128 +0,0 @@
-html, body{
```
Comment (davidmeirlevy) [inline shay-geva/style.css]: probably need to rebase better, because you removed this file.

File: students.html [hunk 4]
```diff
@@ -1,10 +1,132 @@
...
-	<meta charset="utf-8">
-	<title>The HTML5 Herald</title>
+  <meta charset="utf-8">
+  <title>The HTML5 Herald</title>
+  <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
-<h1>The Students</h1>
+  <div class='container'>
```
Comment (davidmeirlevy) [inline students.html]: in HTML, we prefer double-quotes ("value") around attributes values.

**Review by davidmeirlevy — CHANGES_REQUESTED** [review]

**Comment by anaalamed** [comment]
could you please take a look and tell me if now it's ok  @davidmeirlevy

**Comment by davidmeirlevy** [comment]
@anaalamed great work.
for the next time, try to write a better title for your PR.
the PR title isn't your name, but the changes you made. in your case, should be something like: `students list - information and styles`

**Pull request merged**